lib/Conversion/TorchToStablehlo/Pooling.cpp

```diff
@@ -35,7 +35,7 @@ static Value createInitialValueForAtenPoolingOp(Operation *op, Type elementTy,
                                                 PatternRewriter &rewriter) {
   auto constType = RankedTensorType::get({}, elementTy);
   // Avg pooling
-  if (isa<AtenAvgPool1dOp, AtenAdaptiveAvgPool2dOp, AtenAvgPool2dOp, AtenCumsumOp>(op)) {
```

Comment:
Is there a reason for adding it here? Given the decomposition, support for `AtenAvgPool1dOp` should be all that is needed

Reply:
You are right, I just removed it from the file